### PR TITLE
Automatically port System.Text.Json.Utf8JsonWriter

### PR DESCRIPTION
**Pull request by carlossanlop**
New batch.

### CODE REVIEW

File: xml/System.Text.Json/Utf8JsonWriter.xml [hunk 1]
```diff
@@ -132,7 +134,7 @@ In the case of a <xref:System.IO.Stream>, this property indicates how much data
       </ReturnValue>
       <Docs>
         <summary>Gets the number of bytes written by the <see cref="T:System.Text.Json.Utf8JsonWriter" /> so far that have not yet been flushed to the output and committed.</summary>
-        <value>To be added.</value>
+        <value>A number representing the number of bytes written so far that have not yet been flushed to the output and commited.</value>
```
Comment (carlossanlop) [inline xml/System.Text.Json/Utf8JsonWriter.xml]:
```suggestion
        <value>The number of bytes written so far by the <see cref="T:System.Text.Json.Utf8JsonWriter" /> that have not yet been flushed to the output and committed.</value>
```

File: xml/System.Text.Json/Utf8JsonWriter.xml [hunk 2]
```diff
@@ -313,7 +323,7 @@ In the case of <xref:System.IO.Stream>, this writes the data to the stream and f
       </ReturnValue>
       <Docs>
         <summary>Gets the custom behavior when writing JSON using this instance, which indicates whether to format the output while writing and whether to skip structural JSON validation.</summary>
-        <value>To be added.</value>
+        <value>One of the enumeration values from <see cref="T:System.Text.Json.JsonWriterOptions" />.</value>
```
[the remark this answers is not in the document]
Comment (carlossanlop) [reply xml/System.Text.Json/Utf8JsonWriter.xml]: Good catch, thank you!

**Comment by carlossanlop** [comment]
The build finished, @rpetrusha. Is this good to merge?

**Comment by rpetrusha** [comment]
Yes, this is ready to merge, @carlossanlop. I'll do it now.

**Pull request merged**